client/my-sites/site-settings/composing/publish-confirmation.jsx

```diff
@@ -87,6 +106,7 @@ export default connect(
 			siteId,
 			fetchingPreferences: isFetchingPreferences( state ),
 			publishConfirmationEnabled: isConfirmationSidebarEnabled( state, siteId ),
+			showPublishFlow: getSelectedEditor( state, siteId ) === 'gutenberg',
```

Comment:
In other instances where `getSelectedEditor()` is being used, I see it used together with `isGutenbergEnabled()` or `isCalypsoifyGutenbergEnabled()`. @Copons How should it be used ideally? Would these other checks be necessary here?

Reply:
Good question!
I'm trying to remember all the nuances of those selectors, but it's really hard... 😅 

There might be edge cases where users are using Guten in wp-admin on site types that Calypso doesn't support Guten for (e.g. Jetpack and VIP?).
This means that someone might be using Guten in wp-admin and Classic in Calypso. They should still be able to control the publish flow for Classic Calypso.

I'd say let's keep the additional check in place just to be sure and avoid possible UX inconsistencies across site types and user preferences, as combinations are too many to be reliably tested:
`showPublishFlow: isGutenbergEnabled( state, siteId ) && getSelectedEditor( state, siteId ) === 'gutenberg'`